Jenkinsfile

```diff
@@ -49,9 +49,7 @@ pipeline {
 
     triggers {
         cron(env.BRANCH_NAME == 'master' ? 'TZ=America/Toronto\n0 0 * * *\n' : '' +
-             env.BRANCH_NAME == 'release/1.2' ? 'TZ=America/Toronto\n0 12 * * *\n' : '' +
-             env.BRANCH_NAME == 'weekly-testing' ? 'H 0 * * 6' : '' +
-             env.BRANCH_NAME == 'weekly-testing-1.2' ? 'H 0 * * 6' : '')
+             env.BRANCH_NAME == 'weekly-testing' ? 'H 0 * * 6' : '')
```

Comment:
I wonder if this should be more generalized and driven as a property of the branch's Jenkinsfile. You could imagine that at any point in time that we could have a number of branches in various levels of activity and this change is really a reflection of the release/1.2 branch's (low) activity level at the moment. But at some point in the future, say when a 1.2.1 is being prepared the activity level could go back up and switching it back to pr with nightly runs might again become appropriate. Other branches could be changing their activity levels at different times also.

Having defined activity levels and defined _cron_ schedules for each activity level and having the test tag setting coordinated with the _activity level_ indicator makes this a much easier, less-co-ordination needed activity in the future, for this branch and future branches.